### ISSUE TITLE
Connection issues

### ISSUE THREAD

**Issue by djungelola** [meatpiHQ/wican-fw#110]
When returning home with my Hyundai Kona , i see that the meatpi becomes green (and blue) but no data is delivered to HA.
In MQTT explorer i see that TX requests are sent, but no RX responses. I can log in to the meatpi via station AP on my local wifi network. It gets the battery data and says connected.

If i reboot the meatpi in the GUI everything starts to work directly, but i would like to avoid this reboot every time i come home.
FW: 2.92
HW: v3.00_obd

Seems like it doesnt initialize the CAN communication correctly.

Br Ola

**Comment by meatpiHQ** [comment]
@djungelola can you please post a screenshot of you device configuration page? Do you have the CAN bitrate set toe Auto?

**Comment by djungelola** [comment]
![image](https://github.com/meatpiHQ/wican-fw/assets/6004620/88884041-2826-4147-8d33-5e7360d1e183)

**Comment by djungelola** [comment]
Yes, thats correct. Car is on. Pulling out and reinsert the meatpi makes it start working as well.
Another thing that get it working is to connect via Carscanner, after that it starts responding as well.

**Comment by djungelola** [comment]
Is it possible to pull some logs from the device?

**Comment by meatpiHQ** [comment]
@djungelola yes, but I have to send you a special firmware If you're interested?

I'm still unable to replicate your issue. Does this happen if you reboot your router or only if the device is away from wifi for a long period of time?

**Comment by djungelola** [comment]
Sure, please send it.
Havent tested to reboot the router. What  i have observed so far is that i am away for a longer time this happens.
Will do a quick test now just driving away for 10-15 minutes.

**Comment by djungelola** [comment]
So after a 10 min test it started to send data as soon as i got home as expected.

**Comment by meatpiHQ** [comment]
@djungelola Here you go. Sorry for the delay. You can connect to USB are read the logs using putty.

[wican-fw_v298_hv300_dbg.zip](https://github.com/meatpiHQ/wican-fw/files/14872219/wican-fw_v298_hv300_dbg.zip)
